external-builds/pytorch/smoke-tests/run_smoke_tests.sh

```diff
@@ -13,7 +13,7 @@ echo 'List the pip packages'
 pip list
 
 echo 'Install pytest'
-PIP_BREAK_SYSTEM_PACKAGES=1 pip install --no-index --find-links=/wheels pytest;
+PIP_BREAK_SYSTEM_PACKAGES=1 pip install pytest --no-cache-dir --find-links=/wheels --index-url=https://pypi.org/simple
```

Comment:
Why these changes here? This shouldn't need to jump through so many hoops to just install pytest. I think this entire bash script shouldn't need to exist at all though. All that matters for a workflow (or developer running it) is:

```python
pip install [path to the wheel test]
pip install -r requirements-tests.txt
pytest ...
```

I'd keep testing changes out of this PR for now and only focus on getting the wheel _build_ working in github actions.